app/views/shared/_items.html.erb

```diff
@@ -0,0 +1,16 @@
+<% @items.each do |item| %>
+  <hr><div class="item" id=<%= item.title %>>
+    <div id="description">
```

Comment:
@rshea303 this is where you can add the div for the picture within the first "#item" div